pytensor/tensor/shape.py

```diff
@@ -541,9 +544,7 @@ def c_code_cache_version(self):
 
 def specify_shape(
     x: Union[np.ndarray, Number, Variable],
-    shape: Union[
-        int, List[Union[int, Variable]], Tuple[Union[int, Variable]], Variable
-    ],
+    shape: Union[ShapeValueType, List[ShapeValueType], Tuple[ShapeValueType]],
```

Comment:
The tuple probably needs ellipsis now that I think about it 🤔

Reply:
damn, you're right.
Will you take care?
Was the accidental find&replace already fixed? (let's do both in one pr)